docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+networks: 
+  chapter:
+services:
+  app:
+    image: node:10-alpine
+    command: 'npm run dev'
+    volumes:
+      - .:/usr/chapter/
+    working_dir: /usr/chapter/
+    ports:
+      - '8000:8000'
+  speccy:
+    image: node:10-alpine
+    command: 'npm run speccy:watch'
+    volumes:
```

Comment:
Recommend creating a [top-level `volume`](https://docs.docker.com/compose/compose-file/#volumes) and having it referenced in the services, instead of declaring it twice.

Reply:
Good idea!

Reply:
Also, feel free to drop a PR in this branch. That'd be helpful